activedirectory.sgnl.yaml

```diff
@@ -0,0 +1,229 @@
+description: "Active Directory as a System of Record"
+hostname: "{{Input Required: Hostname}}"
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: "ActiveDirectory-1.0.0"
+# Example Config:
+#{
+#    "baseDN":"dc=org,dc=example,dc=io",
+#    "certificateChain": ". . . base64 encoded certificate chain here . . ",
+#    "filters":{
+#         "User":"(&(objectClass=user)(objectCategory=user)(distinguishedName=*))",
+#         "Group":"(&(objectClass=group)(objectCategory=group)(distinguishedName=*))"
+#    }
+#}
+adapterConfig: "ewogICAgImJhc2VETiI6ImRjPW9yZyxkYz1leGFtcGxlLGRjPWlvIiwKICAgICJjZXJ0aWZpY2F0ZUNoYWluIjogIi4gLiAuIGJhc2U2NCBlbmNvZGVkIGNlcnRpZmljYXRlIGNoYWluIGhlcmUgLiAuICIsCiAgICAiZmlsdGVycyI6ewogICAgICAgICJVc2VyIjoiKCYob2JqZWN0Q2xhc3M9dXNlcikobG9nb25Db3VudD0wKSkiCiAgICB9Cn0="
```

Comment:
```suggestion
adapterConfig: "ewogICAgImJhc2VETiI6ICJ7e0lucHV0IFJlcXVpcmVkOiBCYXNlRE59fSIsCiAgICAiY2VydGlmaWNhdGVDaGFpbiI6ICJ7e0lucHV0IFJlcXVpcmVkOiBJZiBMREFQUyBpcyBlbmFibGVkfX0iLAogICAgImZpbHRlcnMiOiB7CiAgICAgICAgIlVzZXIiOiAiKCYob2JqZWN0Q2xhc3M9dXNlcikpIiwKICAgICAgICAiR3JvdXAiOiAiKCYob2JqZWN0Q2xhc3M9Z3JvdXApKSIKICAgIH0KfQ=="
```

Reply:
The default config must be production ready. Add `{{Input Required}}` placeholders wherever you expect user configurable input.